File: Selenium/Project/Activity2.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

public class Activity2 {

    WebDriver driver = new FirefoxDriver();

    @BeforeMethod
    public void openBrowser() throws InterruptedException {
        WebDriverManager.firefoxdriver().setup();
        driver.get("https://alchemy.hguy.co/jobs/");
        driver.manage().window().maximize();
        Thread.sleep(1000);
    }
    @Test
    public void verifyTitle(){
        WebElement heading=driver.findElement(By.xpath("//h1[text()='Welcome to Alchemy Jobs']"));
        if(heading.getText().equals("Welcome to Alchemy Jobs"))
        {
            System.out.println("Headline matches  :" +heading.getText());
            driver.quit();
        }
        else{
            System.out.println("Headline doesn't matches:" +heading.getText());
        }
    }
}

File: Selenium/Project/Activity2.py
import time

import pytest
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC


@pytest.fixture(scope="session")
def setup(request):


    driver = webdriver.Firefox()


    session = request.node


    for item in session.items:
        cls = item.getparent(pytest.Class)
        setattr(cls.obj,"driver",driver)

    # Execute the test method
    yield

    # Close the browser once the test has ended
    request.addfinalizer(driver.close)

@pytest.mark.usefixtures("setup")
class TestSite:

    def test_home_page(self):
        self.driver.get("https://alchemy.hguy.co/jobs/")
        self.driver.maximize_window()
        time.sleep(100)
        headline=self.driver.find_element(By.XPATH,"//h1[text()='Welcome to Alchemy Jobs']")
        actualheadline="Welcome to Alchemy Jobs"
        if headline.text==actualheadline:
            print("\nHeadline Matched")
        else:
            print("Headline is not Matched")

